pkg/scheduler/framework/runtime/framework_test.go

```diff
@@ -134,6 +134,14 @@ func (pl *PluginNotImplementingScore) Name() string {
 	return pluginNotImplementingScore
 }
 
+func newTestPlugin(injArgs runtime.Object, f framework.Handle) (framework.Plugin, error) {
+	var inj injectedResult
+	if err := DecodeInto(injArgs, &inj); err != nil {
```

Comment:
Since you are not using any plugin config, this is unnecessary. Just return `TestPlugin{name: testPlugin}`.